### PR TITLE
fix: #1145 fixed status completed in AllWaysPage

### PR DESCRIPTION
**Pull request by AKiroi**
image

### CODE REVIEW

File: mw-webapp/src/dataAccessLogic/DTOToPreviewConverter/wayPlainDTOToWayPreview.ts [hunk 1]
```diff
@@ -9,7 +9,7 @@ import {WayPreview} from "src/model/businessModelPreview/WayPreview";
  */
 export const wayPlainDTOToWayPreview = (wayDTO: SchemasWayPlainResponse): WayPreview => {
   const status = getWayStatus({
-    status: wayDTO.isCompleted ? "Completed" : null,
+    status: wayDTO.isCompleted ? "completed" : null,
```
Comment (Ekaterina1994) [inline mw-webapp/src/dataAccessLogic/DTOToPreviewConverter/wayPlainDTOToWayPreview.ts:12]: let's don't use just string 

we have enum WayStatus.completed

File: mw-webapp/src/dataAccessLogic/DTOToPreviewConverter/wayPlainDTOToWayPreview.ts
Comment (Ekaterina1994) [inline mw-webapp/src/dataAccessLogic/DTOToPreviewConverter/wayPlainDTOToWayPreview.ts:1]: Please check do we use string "completed" or other statuses anywhere in the code/ Let's update it too) And check do it work correctly)

**Review by Ekaterina1994 — CHANGES_REQUESTED** [review]

File: mw-webapp/src/dataAccessLogic/DTOToPreviewConverter/wayPlainDTOToWayPreview.ts (hunk 1 above)
Comment (AKiroi) [reply mw-webapp/src/dataAccessLogic/DTOToPreviewConverter/wayPlainDTOToWayPreview.ts]: done

File: mw-webapp/src/dataAccessLogic/DTOToPreviewConverter/wayPlainDTOToWayPreview.ts
Comment (AKiroi) [reply mw-webapp/src/dataAccessLogic/DTOToPreviewConverter/wayPlainDTOToWayPreview.ts:1]: fixed

**Review by Ekaterina1994 — APPROVED** [review]

**Review by tritonJS826 — APPROVED** [review]

**Pull request merged**